Automattic/Gravatar-SDK-Android
# Change the skeleton rounder corners value to 50 percent

Closes #199

### Description

So the TextMeasurer works fine but we do use slightly different sizing compared to Figma that results in those visual differences of the Skeleton objects. 

While at it I've noticed the rounded corners aren't flexible enough to accomodate taller Skeleton objects so I've updated them to a percentage value. 

Here's the comparison from the screenshots test
<img width="2497" alt="Screenshot 2024-07-04 at 12 41 52" src="https://github.com/Automattic/Gravatar-SDK-Android/assets/12660801/2c048bf0-6042-46db-9c90-e6c79b83dc42">


### Testing Steps

1. Start the demo app
2. Go to the profile tab
3. Tap the Skeleton button
4. Verify the corner radius works for all skeleton objects
